scheduler/views.py

```diff
@@ -1,2 +1,32 @@
+from datetime import timedelta
+
 from django.conf import settings
+from django.utils import timezone
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import authentication, permissions
+
+from .models import ReminderSchedule
+
+class ReminderCreate(APIView):
+    queryset = ReminderSchedule.objects.all()
+
+    def post(self, request, *args, **kwargs):
+        try:
+            recipient_id = request.data['contacts'][0]['wa_id']
+        except KeyError:
+            status = 400
+            message = {"contacts.0.wa_id": ["This field is required."]}
+            return Response(message, status=status)
+        
+        # Default to 23 hours but allow us to overwrite it
+        delay = int(request.GET.get('hour_delay', 23))
+        scheduled_for = timezone.now() + timedelta(hours=delay)
+        ReminderSchedule.objects.create(
+            schedule_time=scheduled_for,
+            recipient_id=recipient_id,
+            sender_id=settings.TURN_NUMBER,
```

Comment:
Hmm, if this is just the same on every row, maybe it's not all that useful to be storing in the database?

Reply:
I agree, not at the moment. At the time I was aiming to make the model a bit more reusable, so that perhaps we could use it for other channels. But we'd have to add new fields for that to work anyway, so may as well cross that bridge when we get to it.